client/src/components/desktop/ResourcesFilter.js

```diff
@@ -67,25 +67,22 @@ function ResourcesFilter(props) {
   return (
     <div className="resources-filter">
       <Dropdown
-        className="dropdown"
-        overlay={radio('Cost', costs, costSelected, false)}
```

Comment:
i think this is getting rid of albert's changes, although i guess it will still evaluate to false since it will be undefined.